src/environments/environment.ts

```diff
@@ -33,6 +33,14 @@ export const ROLES = {
     value: 'time-tracker-tester',
   },
 };
+
+/*
+This variable is used by the ng-autocomplete component used in these files:
+- details-fielfs.component.ts
```

Comment:
It's not necessary to put all files where variable is used, remove this

Reply:
This was a suggestion made by Sandro